docs/platforms/kotlin-multiplatform/swift-package-manager.mdx

```diff
@@ -1,115 +1,16 @@
-```
-
-Next, install the Kotlin Multiplatform SDK and setup your Apple targets by adding the following to your `build.gradle.kts` file in your shared module:
-
-```kotlin {filename:shared/build.gradle.kts}
-repositories {
-    mavenCentral()
-}
-
-kotlin {
-  listOf(
-    iosX64(),
-    iosArm64(),
-    iosSimulatorArm64()
-  ).forEach {
-    it.binaries.framework {
-        baseName = "shared"
-        isStatic = true // currently available only as static framework
-    }
-  }
-
-  sourceSets {
-    val commonMain by getting {
-      dependencies {
-        implementation("io.sentry:sentry-kotlin-multiplatform:{{@inject packages.version('sentry.kotlin.kmp', '0.0.1-alpha.2') }}")
-      }
-    }
-
-    // Apple targets:
-    val iosMain by getting {
-      dependsOn(commonMain)
-    }
-
-    // Other targets...
-  }
-}
-```
-
-## Running Tests
-
-If you configured the Sentry Kotlin Multiplatform SDK with Swift Package Manager and try running tests, it won't work out of the box and you'll encounter the following error: `ld: framework 'Sentry' not found`.
-
-Follow these steps for the workaround:
-
-### 1. Download the Sentry xcframework
-
-Choose the release based on the version specified in the version compatibility table, download the `Sentry.xcframework.zip`, and unzip it.
-
-### 2. Create Frameworks directory and insert Sentry.framework
-
-Create a `/Frameworks` directory in the directory where the `test.kexe` resides. Put the `Sentry.framework` into it. (The `Sentry.framework` can be found inside of the `ios-arm64_x86_64-simulator`.)
-The `test.kexe` will usually reside in `build/bin/iosSimulatorArm64/debugTest`.
-
-### 3. Add the linker options
-
-```kotlin {filename:shared/build.gradle.kts}
-  listOf(
-      iosX64(),
-      iosArm64(),
-      iosSimulatorArm64(),
-  ).forEach {
-      it.binaries.framework {
-          baseName = "shared"
-          isStatic = true
-      }
-      it.compilations.all {
-          if (compilationName == "test" && target.platformType == KotlinPlatformType.native) {
-              compilerOptions.configure {
-                  freeCompilerArgs.add("-linker-options")
-                  freeCompilerArgs.add("-F/your/path/Carthage/Build/Sentry.xcframework/ios-arm64_x86_64-simulator/")
-              }
-          }
-      }
-  }
-```
+Integrating the Sentry Kotlin Multiplatform SDK while using Swift Package Manager might seem unconventional at first, especially if you're accustomed to using Swift Package Manager for all your dependencies. However, you can achieve a smooth integration by utilizing the Kotlin Cocoapods plugin specifically for compiling and building the shared framework, and then continue to manage all other dependencies with Swift Package Manager as usual.
 
-### 4. Run the tests
+Follow the steps on our primary installation method [CocoaPods](/platforms/kotlin-multiplatform/#install) to install the Sentry Kotlin Multiplatform SDK.
```

Comment:
The link from that [getting started page](https://sentry-docs-git-feat-kmp-improve-spm-docs.sentry.dev/platforms/kotlin-multiplatform/) also links back here... 

Can we update this guide with more useful install instructions or is it actually just the same as the getting started page?